OpenRA.Mods.Cnc/Traits/Infiltration/InfiltrateForDecoration.cs

```diff
@@ -48,8 +48,7 @@ void INotifyInfiltrated.Infiltrated(Actor self, Actor infiltrator, BitSet<Target
 
 		protected override bool ShouldRender(Actor self)
 		{
-			return self.World.RenderPlayer == null || infiltrators.Any(i =>
-				Info.ValidStances.HasStance(i.Stances[self.World.RenderPlayer]));
+			return self.World.RenderPlayer == null || infiltrators.Any(i => Info.ValidStances.HasStance(i.RelationshipWith(self.World.RenderPlayer)));
```

Comment:
Is the `self.World.RenderPlayer == null` necessary now?

Reply:
This would be a behavioural change for when `ValidStances` doesn't contain `Ally` (or `Neutral` depending on `NonCombatant`), so depends on if we want that.

Reply:
Removed it.